tensorflow_encrypted/protocol/pond.py

```diff
@@ -434,6 +434,29 @@ def transpose(self, x):
 
         return x_t
 
+    # see https://www.tensorflow.org/api_docs/python/tf/strided_slice for documentation on
+    # the arguments
+    def strided_slice(self, x, *args, **kwargs):
+        node_key = ('strided_slice', x)
+        x_t = _nodes.get(node_key, None)
```

Comment:
this `_t` naming was for transpose; maybe replace them with `_slice` for clarity?